apps/game_backend/priv/repo/migrations/20240513180434_add_more_match_results_data.exs

```diff
@@ -0,0 +1,33 @@
+defmodule GameBackend.Repo.Migrations.AddMoreMatchResultsData do
+  use Ecto.Migration
+
+  def change do
+    alter table(:arena_match_results) do
+      add :damage_done, :integer
+      add :damage_taken, :integer
+      add :health_healed, :integer
+      add :killed_by, :string
+      add :killed_by_bot, :boolean
+      add :duration_ms, :integer
+    end
+
+    ## This is to allow us to later set `null: false` on all the new columns
+    ## without having to set a default value when adding the column
+    ## Having a default value in the DB could lead to us having a hidden bug or wrong values if
+    ## it where to be used by accident
+    query = """
```

Comment:
Help me understand this 😄 . This is only for previous records, right?

Reply:
Yep exactly, I want to enforce `null: false` on this columns, but I can't do it if they are previous records unless I put a default value which I don't want to as mentioned in the comment

So this query essentially sets some one-time default values for existing records so then after we can enforce `null: false` on the new columns

Reply:
Thanks for the explanation!